spec/integration/manifest.spec.yml

```diff
@@ -225,13 +225,21 @@ spec:
             enum:
               - action
               - next_step
+          appId:
+            description: The identifier of the Kibana app to navigate to. Set this field when `internal` is true.
+            type: string
           content:
             description: Link description
             type: string
+          internal:
+            description: Set to true for internal kibana links, false for external links. Defaults to false.
+            type: boolean
+            default: false
         required:
         - title
         - url
         - type
+        - internal
```

Comment:
Do we need to make it required? Maybe we can assume that this is an external link if internal is not set?

Reply:
yes I was unsure about this, I can remove since I added the default